pkgs/development/compilers/lingua-franca/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, stdenv, jdk11_headless, ... }:
+stdenv.mkDerivation {
+  name = "lfc";
+  version = "0.1.0";
+
+  src = fetchGit {
+    url = "https://github.com/revol-xut/lingua-franca-nix-releases.git";
+    rev = "11c6d5297cd63bf0b365a68c5ca31ec80083bd05";
+    ref = "master";
```

Comment:
We can't use ref master but also rev should be preferred. 

```suggestion
```

Also this is missing sha256.

Reply:
removed ref and added sha256